src/directives/scrollspy/scrollspy.class.js

```diff
@@ -100,8 +107,14 @@ function typeCheckConfig (componentName, config, configTypes) /* istanbul ignore
 
       if (!new RegExp(expectedTypes).test(valueType)) {
         warn(
-          componentName + ': Option "' + property + '" provided type "' +
-                    valueType + '", but expected type "' + expectedTypes + '"'
+          componentName +
+            ': Option "' +
+            property +
+            '" provided type "' +
+            valueType +
+            '", but expected type "' +
+            expectedTypes +
+            '"'
```

Comment:
We could probably switch this to use interpolated string:
```js
warn(
  `${componentName}: Option "${property}" provided type "${valType}" but expected type "${expectedTypes}"`
)
```

Although this may exceed maximum ling length, but it is more readable.